rust/protocol/src/identity_key.rs

```diff
@@ -72,20 +89,29 @@ impl From<IdentityKey> for PublicKey {
     }
 }
 
+/// The private identity of a user.
+///
+/// This cryptographic identity is used to sign pre-keys in [PreKeySignalMessage], to conceal
+/// sealed-sender [crate::sealed_sender::UnidentifiedSenderMessage]s, or to initiate the key
+/// derivation function for a [SessionRecord].
```

Comment:
I'm again unhappy with including the specific uses. These are some ways identity key pairs are used *today* in *this* crate, but Signal uses them for lots of other things too at the app level, and we might use them more for other things in the future.

Reply:
Ok, thanks for that feedback! I think my motivation for including the specifics was more just as my own notes on figuring out the codebase, and I'll avoid including these in the future.